docs/configuration-examples.md

```diff
@@ -0,0 +1,97 @@
+# Configuration examples
+
+Examples of configuration TOML files, you can start with one of this and modify to meet your needs.
+
+## [rest.bitcoin.com tech stack](./architecture#1.-rest.bitcoin.com-tech-stack)
+
+```toml
+# Example rest.bitcoin.com tech stack TOML config file
+
+network = "mainnet"
+rest_service = "bchrest"
+exposed_ports = true
+docker_network = "bch-devsuite-network"
+
+[local_node]
+node = "bu"
+wait_time = 6
+
+[slp]
+username = "actorforth"
+password = "123ldsfoijqwerj"
+
+[bch_rpc_conf]
+host = "http://bch-node"
+port = "8332"
+username = "actorforth"
+password = "BWrzap0bqMjezoeHtOzgOOcUgzkxaL6w"
+```
+
+---
+
+## [Chris Troutner JS tech stack](./architecture#2.-chris-troutner-js-tech-stack)
+
+```toml
+# Example Chris Troutner JS Tech Stack TOML config file
+
+network = "mainnet"
+rest_service = "bchapi"
+exposed_ports = true
+docker_network = "bch-devsuite-network"
+
+[local_node]
+node = "bu"
+wait_time = 6
+
+[slp]
+username = "actorforth"
+password = "123ldsfoijqwerj"
+
+[bch_rpc_conf]
+host = "http://bch-node"
+port = "8332"
+username = "actorforth"
+password = "BWrzap0bqMjezoeHtOzgOOcUgzkxaL6w"
+```
+
+---
+
+## SmartBCH Single Node Private Testnet
+
+```toml
+network = "regtest"
+exposed_ports = true
+docker_network = "bch-devsuite-network"
+
+[local_node]
+node = "bu"
+regtest_blocks = "200"
+wait_time = 6
```

Comment:
Need a comment here that this is a delay before running initializes command.
Need to be high if you run on a slow system.

Feature request:
Or better yet have polling for node readiness